cni/ipam/ipam.go

```diff
@@ -61,22 +62,22 @@ func (plugin *ipamPlugin) Start(config *common.PluginConfig) error {
 	// Initialize base plugin.
 	err := plugin.Initialize(config)
 	if err != nil {
-		log.Printf("[cni-ipam] Failed to initialize base plugin, err:%v.", err)
+		log.Logger.Error("[cni-ipam] Failed to initialize base plugin.", zap.String("error", err.Error()))
```

Comment:
using zap.Any everywhere for error. can we use same here?

Reply:
done